### PR TITLE
Package pymartini

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by davidbrochart** [comment]
@kylebarron do you agree to be a maintainer of this package?

**Comment by kylebarron** [comment]
Yes I agree

File: recipes/pymartini/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: {{ build }}
+  skip: true  # [py<36 or python_impl=="pypy"]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (synapticarbors) [inline recipes/pymartini/meta.yaml:21]: does this need the `cxx` compiler or is just a c-compiler sufficient. I didn't see any c++ in the repo

File: recipes/pymartini/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e586fb492459d6ab80f442929b622d1567d2dc77bcffcdf49ac5146567b22944
+
+build:
+  number: {{ build }}
```
Comment (synapticarbors) [inline recipes/pymartini/meta.yaml:14]: You can just explicitly put the build number here and not use the jinja templating. The regro bot will handle it correctly.

File: recipes/pymartini/meta.yaml (hunk 1 above)
Comment (davidbrochart) [reply recipes/pymartini/meta.yaml:21]: Yes, I guess a C compiler is enough.

File: recipes/pymartini/meta.yaml (hunk 2 above)
Comment (davidbrochart) [reply recipes/pymartini/meta.yaml:14]: :+1:

**Pull request merged**